src/app/core/services/admin.service.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-console */
+'use strict';
+
+import firebase from 'firebase';
```

Comment:
no dependencies outside the angular

Reply:
AngularFire was moved into Angular, but the import of Firebase was left because there is no another possibility to use 'firebase' object, which we need for initializing references to storage, database and using cloud functions.